ci/jenkins/jobs/projects.yaml

```diff
@@ -325,3 +325,33 @@
           started_status: null
           wrappers: []
           publishers: []
+      - '{name}-{test_name}-for-eks':
+          test_name: conformance-net-policy
+          node: antrea-cloud
+          description: 'This is EKS conformance and network policy test for antrea.'
+          builders:
+          - shell: |-
+              #!/bin/bash
+              ./ci/test-conformance-eks.sh
+          triggers:
+          - timed: H H * * *
+          publishers:
+          - archive:
+              allow-empty: true
+              artifacts: eks-test.log
+              case-sensitive: true
+              default-excludes: true
+              fingerprint: false
+              only-if-success: false
+          - trigger-parameterized-builds:
+            - project:
+              - cloud-cleanup-eks-testbed
+              property-file: 'ci_properties.txt'
+          wrappers:
+          - credentials-binding:
+            - text:
+                credential-id: e7c8a876-9dbc-4c8c-82fa-7031fb7bb0cd # Jenkins secret that stores aws access key
```

Comment:
When creating secret, Jenkins allow the admin the specify an arbitrary string as ID. So maybe we can configure fixed string IDs like `aws-access-key` and `aws-secret-key`.

Reply:
Sure. When I created the secrets I was using Chrome and it did not actually prompt me to change the ID. Updated